samples/net/azure_iot_hub/src/main.c

```diff
@@ -323,7 +323,7 @@ static void direct_method_handler(struct k_work *work)
 		},
 		.status = 200,
 		.payload.ptr = response,
-		.payload.size = sizeof(response) - 1,
+		.payload.size = strlen(response),
```

Comment:
We could avoid allocating a pointer by doing
```
static const char response[] = "{\"it\":\"worked\"}";
```
and then we could use the built time sizeof instead of runtime strlen.